manifests/01_namespace.yaml

```
@@ -10,6 +10,6 @@ metadata:
     capability.openshift.io/name: "marketplace"
   labels:
     openshift.io/cluster-monitoring: "true"
-    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce: privileged
```

Have we checked to see if this could be `baseline` instead of `privileged` and still support `legacy` catalog sources?

```suggestion
    pod-security.kubernetes.io/enforce: baseline
```

---

Also, let's add the following?
```
pod-security.kubernetes.io/audit: restricted
pod-security.kubernetes.io/warn: restricted
```